disco_aws_automation/disco_vpc.py

```diff
@@ -7,64 +7,65 @@
 import logging
 import random
 
-import time
 from ConfigParser import ConfigParser
 
-import boto
-import boto.ec2
-from boto.vpc import VPCConnection
 from boto.exception import EC2ResponseError
 import boto3
+
 from netaddr import IPNetwork, IPSet
 
 from disco_aws_automation.network_helper import calc_subnet_offset
-from . import read_config, normalize_path
+from . import normalize_path
 
-from .resource_helper import (
-    keep_trying,
-    wait_for_state,
-    wait_for_state_boto3
-)
+from .resource_helper import tag2dict
 from .disco_log_metrics import DiscoLogMetrics
 from .disco_alarm import DiscoAlarm
 from .disco_alarm_config import DiscoAlarmsConfig
 from .disco_autoscale import DiscoAutoscale
 from .disco_constants import CREDENTIAL_BUCKET_TEMPLATE, NETWORKS
 from .disco_metanetwork import DiscoMetaNetwork
+from .disco_vpc_sg_rules import DiscoVPCSecurityGroupRules
+from .disco_vpc_gateways import DiscoVPCGateways
+from .disco_vpc_peerings import DiscoVPCPeerings
 from .disco_elasticache import DiscoElastiCache
 from .disco_sns import DiscoSNS
 from .disco_rds import DiscoRDS
-from .disco_eip import DiscoEIP
 from .disco_elb import DiscoELB
 from .exceptions import (
-    MultipleVPCsForVPCNameError, TimeoutError, VPCConfigError, VPCEnvironmentError, VPCPeeringSyntaxError,
-    VPCNameNotFound, EIPConfigError)
+    MultipleVPCsForVPCNameError, VPCConfigError, VPCEnvironmentError,
+    VPCNameNotFound)
 
 
 CONFIG_FILE = "disco_vpc.ini"
-VGW_STATE_POLL_INTERVAL = 2  # seconds
-VGW_ATTACH_TIME = 600  # seconds. From observation, it takes about 300s to attach vgw
-LIVE_PEERING_STATES = ["pending-acceptance", "provisioning", "active"]
 
 
+# FIXME: pylint thinks too many lines are in the file and it has too many instance arguments
```

Comment:
Good catch. Though we still have a lot of instance arguments :( I'll remove the one about too many lines.